hack/find-godeps.sh

```diff
@@ -23,7 +23,7 @@ function find-deps() {
 	local deps=
 
 	# gather imports from cri-o
-	pkgs=$(cd ${basepath}/${srcdir} && go list -f "{{.Imports}}" . | tr ' ' '\n' | grep -v "/vendor/" | grep ${pkgname} | sed -e "s|${pkgname}/||g")
+	pkgs=$(cd ${basepath}/${srcdir} && go list -f "{{join .Imports "\n"}}" . | grep -v "/vendor/" | grep ${pkgname} | sed -e "s|${pkgname}/||g")
```

Comment:
I can't find a reference to join being available out of the box without being compiled as a template function.